mobs/owl.lua

```diff
@@ -1,46 +1,85 @@
-mobs:register_mob("dmobs:owl", {
-	type = "animal",
-	passive = true,
-	hp_min = 12,
-	hp_max = 22,
-	armor = 130,
-	collisionbox = {-0.3, -0.5, -0.3, 0.2, 0.2, 0.2},
-	visual = "mesh",
-	mesh = "owl.b3d",
-	textures = {
-		{"dmobs_owl.png"},
+mobs:register_mob("mobs_doomed:owl", {
```

Comment:
Please change namespace back to `dmobs:*`
Chaning this namespace here may break compatibility in other mobs depending on this one.

Reply:
I believe you may be looking at an outdated comit, I forced ammended it.

Reply:
ah, fiddle sticks. That is on me, I pushed to the wrong branch. the most up to date commit should work.